easybuild/easyconfigs/s/SYMPHONY/SYMPHONY-5.6.16-foss-2018b.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'SYMPHONY'
+version = '5.6.16'
+
+homepage = 'https://projects.coin-or.org/SYMPHONY'
+description = """SYMPHONY is an open-source solver for mixed-integer linear programs (MILPs) written in C."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.coin-or.org/download/source/SYMPHONY/']
+sources = [SOURCE_TGZ]
+checksums = ['54df3fd953d9a2e4178f10373c6cb8b46a0808569b42adf07aa6d0044b2ab04b']
+
+dependencies = {
+    ('CoinUtils', '2.11.3'),
+    ('Osi', '0.108.5'),
+    ('Clp', '1.17.3'),
+    ('Cgl', '0.60.2'),
+}
+
+sanity_check_paths = {
+    'files': ['bin/symphony', 'lib/libSym.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'lib', 'include/coin'],
```

Sanity checks should be updated as well to cover more files in the installation
```suggestion
    'files': ['bin/symphony', 'lib/libSym.%s' % SHLIB_EXT, 'lib/libOsiSym.%s' % SHLIB_EXT],
    'dirs': ['include/coin', 'lib/pkgconfig', 'share/coin']
```